global/scripts/MyelinMap_BC.sh

```diff
@@ -178,6 +155,13 @@ log_Msg "Resampled BiasField in the native mesh space: ${NativeBiasField}"
 
 # generate bias corrected map in the output mesh space
 NativeBCMap=${NativeFolder}/${Subject}.MyelinMap_BC${RegNameInOutputName}.native.dscalar.nii
+NativeOldBCMap=${NativeFolder}/${Subject}.MyelinMap_oldBC${RegNameInOutputName}.native.dscalar.nii
+# TODO: maybe we should move the file operation outside of this module
+if [[ ! -f "$NativeOldBCMap" ]]; then
```

Comment:
I still think this renaming of the old version shouldn't be in the pipelines code itself.  If this code is run on fresh data, and then run a second time, it will make an _oldBC file that was the first run of the new code.

Reply:
Now I comment out these lines. I agree that this shouldn't be in the official pipeline as it will cause troubles for users and is not explicitly mentioned to them. The comment now is just a reminder to us if we want to generate new _BC and save the old ones.

Reply:
Looks like the lines were deleted, with no remaining comment, but deleting the lines from this script is what I would suggest.  As long as we remember (some other way) to do this rename before running the modified pipeline with the new template on the full dataset.